e2e/create-cli-e2e/tests/init.e2e.test.ts

```diff
@@ -1,9 +1,9 @@
+import { dirname } from 'knip/dist/util/path';
```

Comment:
I am not sure if we should be using knip util there
Did you mean to use `import { dirname } from 'node:path';`?

Reply:
Uuups! Nice catch! ❤️